### ISSUE TITLE
FileRelationship: (fields.W340) null has no effect on ManyToManyField

### ISSUE THREAD

**Issue by hackdna** [refinery-platform/refinery-platform#2611]
* Specific code commit: ae273b9
* Version of the web browser and OS: N/A
* Environment where the error occurred: Vagrant

### Steps to reproduce
```bash
$ manage.py migrate
```

### Observed behavior
```
[vagrant@127.0.0.1:2222] run: /vagrant/refinery/manage.py migrate --noinput
[vagrant@127.0.0.1:2222] out: System check identified some issues:
[vagrant@127.0.0.1:2222] out:
[vagrant@127.0.0.1:2222] out: WARNINGS:
[vagrant@127.0.0.1:2222] out: tool_manager.FileRelationship.file_relationship: (fields.W340) null has no effect on ManyToManyField.
```

### Expected behavior
No warning

**Comment by jkmarx** [comment]
Oh, I thought I resolved that issue in the 1.8 upgrade

**Comment by jkmarx** [comment]
nevermind
